lib/registries/addon/components/registries-recent-list/styles.scss

```diff
@@ -5,6 +5,8 @@
     border-bottom: 1px solid #e0e6ea;
     padding-bottom: 15px;
     padding-top: 15px;
+    padding-left: 50px;
+    padding-right: 50px;
```

Comment:
Applying this to staging2, it looks like these paddings are more appropriate on the `.recent-registries-container` class